test/test_blocks.py

```diff
@@ -197,5 +197,25 @@ def test_integration_mlflow(self):
         demo.close()
 
 
+def test_slider_random_value_config():
+    with gr.Blocks() as demo:
+        gr.Slider(
+            value=11.2, minimum=-10.2, maximum=15, label="Non-random Slider (Static)"
+        )
+        gr.Slider(
+            randomize=True, minimum=100, maximum=200, label="Random Slider (Input 1)"
+        )
+        gr.Slider(
+            randomize=True, minimum=10, maximum=23.2, label="Random Slider (Input 2)"
+        )
+    for component in demo.blocks.values():
+        if "Non-random" in component.label:
+            assert not component.randomize
+        else:
+            assert component.randomize
+    assert all([dep["trigger"] == "load" for dep in demo.config["dependencies"]])
```

Comment:
I think we should also confirm the existence of 2 `load()` dependencies

Reply:
Agreed